pkg/snapshot/v1alpha1/snapshot_template_engine.go

```diff
@@ -15,6 +15,7 @@ limitations under the License.
 */
 
 // This file has snapshot specific implementation of cas template engine
```

Comment:
Can you please delete this comment line.

Reply:
Done